src/Razor/src/rzls/rzls.csproj

```diff
@@ -40,6 +40,7 @@
   <ItemGroup>
     <!-- We need to directly reference the O# language server here because we mark it as PrivateAssets="All" in our language server itself -->
     <PackageReference Include="OmniSharp.Extensions.LanguageServer" Version="$(OmniSharpExtensionsLanguageServerPackageVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="$(MicrosoftExtensionsPackageVersion)" />
```

Comment:
Why was this bit added? This only impacts VSCode is why I ask

Reply:
That was an attempt to fix [this kind of failure](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=6091755&view=logs&j=bf62c3a4-1004-564b-6dcc-6dcb68a53174&t=26d7af73-c5ca-595e-2ae0-d3b785497762).

Reply:
Ah I'm not sure this would do that given it's an isolated assembly but maybe I'm wrong. On semi-recent `main` I had [this](https://github.com/dotnet/razor-tooling/commit/10ac8ed50492bc15b9effc233476c98ff767d393) as an attempt to address